{"uuid":"822a45ab-5185-4a4c-bf86-d58899f5480e","historyId":"4d7dcf6bfb97c7cdd51bbaeddf240418","fullName":"Array Page: The user is presented with error message for invalid code in tryEditor for Max Consecutive Ones","labels":[{"name":"tag","value":"Array"},{"name":"tag","value":"Array27"},{"name":"host","value":"DESKTOP-88CLK0G"},{"name":"thread","value":"7180@DESKTOP-88CLK0G.main(1)"},{"name":"feature","value":"Array Page"},{"name":"story","value":"The user is presented with error message for invalid code in tryEditor for Max Consecutive Ones"},{"name":"suite","value":"Array Page"},{"name":"testClass","value":"The user is presented with error message for invalid code in tryEditor for Max Consecutive Ones"},{"name":"framework","value":"cucumber4jvm"},{"name":"language","value":"java"},{"name":"gherkin_uri","value":"file:///C:/Users/anoop/eclipse-workspaceNew/DS_AlgoProjectNew/src/test/resources/features/05Array.feature"},{"name":"package","value":"src.test.resources.features.05Array_feature.Array Page"}],"links":[],"name":"The user is presented with error message for invalid code in tryEditor for Max Consecutive Ones","status":"passed","stage":"finished","steps":[{"name":"Given  The user is on Question page of Max Consecutive Ones after logged in","status":"passed","statusDetails":{"known":false,"muted":false,"flaky":false},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1689020320305,"stop":1689020320315},{"name":"When  The user enter python code with invalid syntax in tryEditor from sheet \"pythonCode\" and 1 for the question","status":"passed","statusDetails":{"known":false,"muted":false,"flaky":false},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1689020320315,"stop":1689020320718},{"name":"And  The user clicks on run button for array","status":"passed","statusDetails":{"known":false,"muted":false,"flaky":false},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1689020320719,"stop":1689020320817},{"name":"Then  The user should be presented with error message for array","status":"passed","statusDetails":{"known":false,"muted":false,"flaky":false},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1689020320818,"stop":1689020320828}],"attachments":[],"parameters":[{"name":"Sheetname","value":"pythonCode"},{"name":"RowNumber","value":"1"}],"start":1689020320304,"stop":1689020320828}